core/bridges/bridge_type.go

```diff
@@ -11,15 +11,16 @@ import (
 	"time"
 
 	"github.com/smartcontractkit/chainlink/core/assets"
+	"github.com/smartcontractkit/chainlink/core/store/models"
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
 // BridgeTypeRequest is the incoming record used to create a BridgeType
 type BridgeTypeRequest struct {
-	Name                   TaskType     `json:"name"`
-	URL                    WebURL       `json:"url"`
-	Confirmations          uint32       `json:"confirmations"`
-	MinimumContractPayment *assets.Link `json:"minimumContractPayment"`
+	Name                   TaskType      `json:"name"`
```

Comment:
I believe these json tags aren't being used anymore since we are using presenters in the `web` pkg. It could be still used in tests, but maybe look into removing them?

Reply:
I'll audit all of these but the `*Request` ones are still used because they're used to deserialize API requests.